### PR TITLE
Added CSharp examples for taking Screenshot and Element Screenshot in Webdriver docs

### PR DESCRIPTION
**Pull request by idkaryan**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Added examples for taking a normal screenshot and web element screenshot to the WebDriver documentation.

### Motivation and Context

Examples for said feature using CSharp were absent. Added it so it would help someone else out.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (only in the English language as I can't read or understand how to navigate to correct positions for these examples in other languages. Would appreciate if someone copies that over.)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Review by diemol — CHANGES_REQUESTED** [review]
@idkaryan could you please add the code examples for the other files? (es, de, etc...)

**Comment by idkaryan** [comment]
Okay, I did those changes. I'll push them along with this one in a new PR. New to open source contribution so not sure if I can just add that here.

**Comment by idkaryan** [comment]
Hey @diemol. Please let me know if there's any issue with the latest commit. Looks like it could be merged in this PR itself, which is neat. Thanks.

File: docs_source_files/content/webdriver/browser_manipulation.de.md [hunk 1]
```diff
@@ -1143,7 +1143,15 @@ driver.save_screenshot('./image.png')
...
 
 {{< / code-panel >}}
-  {{< code-panel language="csharp" >}} // Code sample not available please raise a PR {{< / code-panel >}}
+  {{< code-panel language="csharp" >}} 
+    using OpenQA.Selenium;
+    using OpenQA.Selenium.Chrome;
+    using OpenQA.Selenium.Support.UI;
+
+    var driver = new ChromeDriver();
+    var screenshot = (driver as ITakesScreenshot).GetScreenshot();
```
Comment (harsha509) [inline docs_source_files/content/webdriver/browser_manipulation.de.md:1153]: I think need to navigate to a url like driver.get("http://www.example.com"); before capturing screenshot

Am not sure how driver builder works in C# if below code
```
 var driver = new ChromeDriver();
```
Opens a blank page, then the above code is valid. Else we need to navigate to URL and take screenshot as other code samples do

File: docs_source_files/content/webdriver/browser_manipulation.de.md [hunk 2]
```diff
@@ -1229,7 +1237,21 @@ ele.screenshot('./image.png')
...
+  {{< code-panel language="csharp" >}}
+    using OpenQA.Selenium;
+    using OpenQA.Selenium.Chrome;
+    using OpenQA.Selenium.Support.UI;
+
+    // Webdriver
+    var driver = new ChromeDriver();
+    
+    // Fetch element using FindElement
+    var webElement = driver.FindElement(By.XPath(@"(//*[@id=""messages""])/div[last() - 1]"));
```
Comment (harsha509) [inline docs_source_files/content/webdriver/browser_manipulation.de.md]: Couple of things,

1. like all other code samples, can we need to navigate to URL (http://www.example.com/) and 
2. like other code samples can we change to driver.FindElement(By.CssSelector("h1"));

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @idkaryan ,

Thanks for the PR. I've suggested few changes can you please look into it

Regards,
Harsha

File: docs_source_files/content/webdriver/browser_manipulation.de.md (hunk 1 above)
Comment (idkaryan) [reply docs_source_files/content/webdriver/browser_manipulation.de.md:1153]: Oh, that's right. This is the code for taking screenshot of an already opened browser which I attached to Webdriver via changing the debugAddress in options. For a new blank page navigating to some url and then screenshotting it, I'll make that change in the code.

**Comment by idkaryan** [comment]
@Harsha509 Made the changes so webdriver opens up a blank page and navigates to http://www.example.com before taking a screenshot and also changed the query for FindElement from Xpath to Css as it was in the other examples. Could you please review it?

**Review by harsha509 — APPROVED** [review]
Thank you @idkaryan !

**Review by diemol — APPROVED** [review]
Thank you @idkaryan!

**Pull request merged**